Podfile.lock

```diff
@@ -7,6 +7,6 @@ DEPENDENCIES:
 SPEC CHECKSUMS:
   OpenTok: 9f0909068d6ed69acffa77a4610b47924926f79e
 
-PODFILE CHECKSUM: d4b63399d3fef5486044af412f4a623ab30c4aa3
+PODFILE CHECKSUM: 3dc4397fcacc9a6d8f8ab5d446cc97ad3f96e91c
```

Comment:
Is there a specific reason to keep a `.lock` in the repo? It might have side effect like `.lock` constraints the OT SDK to 2.9.2. And users never get a chance to use our latest iOS SDK.